README.md

```diff
@@ -52,18 +52,16 @@ import 'clipboard-copy-element'
 
 ## Events
 
-After copying to the clipboard, a [copy][] event is dispatched that can be
-used to notify the user with confirmation, like a tooltip or button highlight.
+After copying to the clipboard, a `clipboard-copied` event is dispatched from
+the `<clipboard-copy>` element:
 
 ```js
-document.addEventListener('copy', function(event) {
-  const button = document.activeElement
+document.addEventListener('clipboard-copied', function(event) {
```

Comment:
I think we settled on event naming that uses the “verbed” (past tense) pattern only if we also publish the cancellable “verb” event before the action. If there is only one, I think the event should be named `clipboard-copy`. Does this make sense?